src/core/client/stream/tabs/Comments/Stream/PostCommentForm/CreateCommentMutation.ts

```diff
@@ -59,14 +77,18 @@ function addCommentToStory(
   // Get stream proxy.
   const streamProxy = store.get(input.storyID);
   const connectionKey = "Stream_comments";
-  const filters = { orderBy: "CREATED_AT_DESC" };
 
-  if (streamProxy) {
-    const con = ConnectionHandler.getConnection(
-      streamProxy,
-      connectionKey,
-      filters
-    );
+  if (input.commentsOrderBy === "CREATED_AT_ASC") {
```

Comment:
`commentsOrderBy` can be null, so we only do this if it's an exact match.

Hence the duplicate `getConnection(...)` rigamarole.